atlp-rwanda/e-comm-team-techsmith-fn
# #185218937 A buyer should be able to add items to their wishlist

## What does this PR do?

This PR it will be Implementing buyer should be able to add items to their wishlist

## Description of Task to be completed?

- Adding Items into Wishlist

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-buyer-should-add-items-in-whishlist#185218937`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185218937](https://www.pivotaltracker.com/story/show/185218937)
